.github/workflows/update-latest-tag.yml

```
@@ -0,0 +1,62 @@
+          docker push guarzo/wanderer-notifier:latest
+          
+      - name: Summary
+        run: |
+          echo "## ✅ Latest Tag Updated" >> $GITHUB_STEP_SUMMARY
+          echo "" >> $GITHUB_STEP_SUMMARY
+          echo "**Updated:** \`guarzo/wanderer-notifier:latest\` → \`${{ github.event.inputs.version_tag }}\`" >> $GITHUB_STEP_SUMMARY
+          echo "" >> $GITHUB_STEP_SUMMARY
+          echo "### Pull Command" >> $GITHUB_STEP_SUMMARY
+          echo "\`\`\`bash" >> $GITHUB_STEP_SUMMARY
+          echo "docker pull guarzo/wanderer-notifier:latest" >> $GITHUB_STEP_SUMMARY
+          echo "\`\`\`" >> $GITHUB_STEP_SUMMARY
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)